src/base/http/responsegenerator.h

```diff
@@ -37,11 +37,9 @@
 
 namespace Http
 {
-    class ResponseGenerator
-    {
-    public:
-        static QByteArray generate(Response response);
-    };
+    QByteArray responseGenerator(Response response);
```

Comment:
I plan to have a class for `Response` in later PR, naming it `Response::generate()` is a bit general, or maybe `Response::toByteArray()`?